src/dimensions.js

```diff
@@ -19,6 +19,11 @@ jQuery.each( { Height: "height", Width: "width" }, function( name, type ) {
 
 				if ( jQuery.isWindow( elem ) ) {
 
+					// $( window ).outerWidth/Height return w/h including scrollbars (gh-1729)
```

Comment:
This can probably be combined into a single return with the value below but I wanted to resolve the status of that comment first.